### ISSUE TITLE
Is there a simple way to include "OUT_DIR" files?

### ISSUE THREAD

**Issue by littzhch** [kornelski/cargo-deb#67]
I try to pack files generated by build.rs into deb package, but I didn't find a comfortable way to fetch files in OUT_DIR using cargo-deb. Is it possible to add a simple way to do this?

**Comment by kornelski** [comment]
There is no good way to do this in `cargo`. The location of `OUT_DIR` changes and it is an implementation detail that is private to Cargo.

**Comment by sowbug** [comment]
@littzhch did you find a solution to this? I want to include the output of [clap_mangen](https://crates.io/crates/clap_mangen) in my package.

**Comment by littzhch** [comment]
> @littzhch did you find a solution to this? I want to include the output of [clap_mangen](https://crates.io/crates/clap_mangen) in my package.

@sowbug No, still unsolved. It seems that a RFC is needed to make `OUTPUT` public. see this: rust-lang/cargo#9858

**Comment by sowbug** [comment]
Thanks. That issue discussion confirms my guess that what we want is
contrary to a design goal. Nobody puts random hash strings in a path to
improve discoverability. :)

For the short term, I did a gross hack, outputting my generated man page to
target/. It'll work until it doesn't, and then cargo deb will complain
about a missing asset, and then I'll hack it again.
